honeycomb-guide/src/project-structure/honeycomb-core.md

```diff
@@ -1 +1,46 @@
 # honeycomb-core
+
+[Documentation](../honeycomb_core/)
+
+--- 
+
+**honeycomb-core** is a Rust crate that provides basic structures and 
```

Comment:
Link to either website or documentation.

Reply:
The link of the header should redirect to crate documentation